src/modules/kazoo/kz_amqp.c

```diff
@@ -846,17 +846,17 @@ void kz_amqp_channel_close(kz_amqp_conn_ptr rmq, amqp_channel_t channel)
 	kz_amqp_error("closing channel",
 			amqp_channel_close(rmq->conn, channel, AMQP_REPLY_SUCCESS));
 }
-#if AMQP_VERSION_MAJOR == 0 && AMQP_VERSION_MINOR < 13
```

Comment:
why are you removing this part?

Reply:
@linuxmaniac This was not supposed to be removed, fixed that

Reply:
not, really. The code is different. Please, put it back as it was.

Reply:
Sorry about the extra work, I overlooked the commit between 5.8.2 and master. That's why this got accidentally removed. It should be as it was now.